README.md

```diff
@@ -1,30 +1,36 @@
 
-Artsy's Design System
+### Storybook 
+[https://palette-storybook.artsy.net](https://palette-storybook.artsy.net) 
+The latest design system and components are available here 🙌🏻 We have a new repo for icons and prefer to use that one ⬇️ 
+Icons will be removed from palette-storybook.artsy.net eventually
 
-## Meta
+### Icons 
+[https://icons.artsy.net](https://icons.artsy.net) use this for icons
 
-- Storybook: [https://palette-storybook.artsy.net](https://palette-storybook.artsy.net)
-- Docs: [https://palette.artsy.net](https://palette.artsy.net)
+### Other links
+- Docs [https://palette.artsy.net](https://palette.artsy.net) To be depracated. It contains out of date Design System and Components. Docs will eventually move to the palette-storybook.artsy.net
 - Docs Github Repo: [https://github.com/artsy/palette-docs](https://github.com/artsy/palette-docs)
+
+## Meta 
+- **@artsy/palette**
+
+- [![CircleCI](https://circleci.com/gh/artsy/palette.svg?style=shield)](https://circleci.com/gh/artsy/palette) [![npm version](https://badge.fury.io/js/%40artsy%2Fpalette.svg)](https://www.npmjs.com/package/@artsy/palette) [![Netlify Status](https://api.netlify.com/api/v1/badges/beb9e8d7-10cc-4a2e-99bb-0d4c6f46db82/deploy-status)](https://app.netlify.com/sites/artsy-palette/deploys)
+
 - Point People: [@dzucconi](https://github.com/dzucconi), [@damassi](https://github.com/damassi)
 
 ## What is Palette?
 
-Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used
-across our digital product portfolio.
+Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used across our digital product portfolio.
 
 ## Does my component belong in Palette?
 
-If the component applies to Artsy as a brand and can/will be used across multiple digital products, then Palette is a great place for it. If it's highly product
-specific then it's best to leave the component where it's used. We can always move things later!
-
-If the above guidance still doesn't give you a good sense of what to do, please join the bi-weekly design systems sync.
+If the component applies to Artsy as a brand and can/will be used across multiple digital products, then Palette is a great place for it. If it's highly product specific then it's best to leave the component where it's used. We can always move things later.
 
 ## How to contribute
 
-If you'd like to add a new component to Palette please create an issue using the component spec template. That'll give both design and engineering a chance
-to peek at the proposal and provide feedback before moving forward.
+If you'd like to add a new component to Palette please create an issue using the component spec template [- link?]. 
+That'll give both design and engineering a chance to peek at the proposal and provide feedback before moving forward.
```

Comment:
This isn't really part of the process any more. I suppose the process would be raising it in the `#design-system` Slack channel